src/test/transaction_tests.cpp

```diff
@@ -780,7 +780,7 @@ BOOST_AUTO_TEST_CASE(test_IsStandard)
     CheckIsStandard(t);
 
     // Disallowed nVersion
-    t.nVersion = -1;
+    t.nVersion = static_cast<uint32_t>(-1);
```

Comment:
ca7090d43fab2013bd396d683f4cf03062666b14

Does this still look meaningful? 
If you want to test a large number, then better check against `TX_MAX_STANDARD_VERSION + 1`?

Reply:
Changed to `std::numeric_limits<uint32_t>::max()`